README.md

```diff
@@ -5,8 +5,14 @@ Index of Marquand auction catalogs.
 ## Local development
 
 1. Run: `npm install`
-2. Start a local server: `npm start`
-3. Visit: [localhost:3456](http://localhost:3456/)
+2. Start a local server: `npm run dev`
+3. Visit: [localhost:5173](http://localhost:5173/)
+
+## Building for deployment
```

Comment:
So this is part of the capistrano deployment, so how can we make that clear? I think it's nice to have here, but I initially thought we needed to do them separately for deployment